feature/experimental/bgp/ate_tests/bgp_tcp_mss_path_mtu/bgp_tcp_mss_path_mtu_test.go

```diff
@@ -0,0 +1,406 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bgp_tcp_mss_path_mtu_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// The testbed consists of ate:port1 -> dut:port1 and
+// dut:port2 -> ate:port2.  The first pair is called the "source"
+// pair, and the second the "destination" pair.
+//
+// 	ATE   Port-1 192.0.2.2 --------  DUT-1 Port-1 192.0.2.1
+//	DUT-1 Port-2 192.0.2.5 --------  DUT-2 Port-1 192.0.2.6
+
+const (
+	peerGrpName1       = "BGP-PEER-GROUP1"
+	peerGrpName2       = "BGP-PEER-GROUP2"
+	dut1AS             = 65501
+	ateAS1             = 65502
+	dut2AS             = 65502
+	plenIPv4           = 30
+	plenIPv6           = 126
+	dut1TcpMssDefault  = uint16(536)
+	dut1Tcp6MssDefault = uint16(1240)
+	dut1IntfMtu        = uint16(5040)
+	dut1TcpMss         = uint16(4096)
+	MTU512             = uint16(512)
+	vlan10             = 10
+	vlan20             = 20
+	ISISInstance       = "DEFAULT"
```

Comment:
I don't think this variable needs to be capitalized/exported

Reply:
updated